app/services/rails_engine_service.rb

```diff
@@ -0,0 +1,25 @@
+class RailsEngineService
```

Comment:
I like that you have one service for all of these. In some apps, this when get really large and unwieldy to manage, but here, that's definitely not an issue so it makes it much easier to maintain in one class.